tests/integration/conftest.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Fixtures for Synapse charm integration tests."""
+
+
+import json
+
+import pytest_asyncio
+from juju.model import Model
+from pytest import Config
+from pytest_operator.plugin import OpsTest
+
+
+@pytest_asyncio.fixture(scope="module", name="server_name")
+async def fixture_server_name() -> str:
+    """Return a server_name."""
+    return "my.synapse.local"
+
+
+@pytest_asyncio.fixture(scope="module", name="model")
+async def fixture_model(ops_test: OpsTest) -> Model:
+    """Return the current testing juju model."""
+    assert ops_test.model
+    return ops_test.model
+
+
+@pytest_asyncio.fixture(scope="module", name="build_charm")
+async def build_charm_fixture(ops_test) -> str:
+    """Build the charm"""
+    charm = await ops_test.build_charm(".")
+    return charm
+
+
+@pytest_asyncio.fixture(scope="module", name="synapse_app")
+async def synapse_app_fixture(
+    build_charm: str,
```

Comment:
Is this type hint right?

Reply:
Well, ops_test.build actually return a pathlib.Path but deploy expects a str so I'm leaving like that.